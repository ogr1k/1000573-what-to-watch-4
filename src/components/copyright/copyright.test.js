import React from "react";
import Copyright from "./copyright.jsx";
import renderer from "react-test-renderer";


it(`Render Copyright`, () => {
  const tree = renderer
    .create(<Copyright />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
