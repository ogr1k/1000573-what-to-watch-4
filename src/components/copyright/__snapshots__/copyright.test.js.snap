// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Copyright 1`] = `
<div
  className="copyright"
>
  <p>
    Â© 
    2020
     What to watch Ltd.
  </p>
</div>
`;
